lua/ulx/modules/sh/cfc_cheese.lua

```diff
@@ -0,0 +1,27 @@
+CFCUlxCommands.cheese = CFCUlxCommands.cheese or {}
+local cmd = CFCUlxCommands.cheese
+
+local CATEGORY_NAME = "Fun"
+local BASE_CHEESE_MODELS = {
+    "models/hunter/triangles/025x025.mdl",
+    "models/hunter/triangles/05x05x05.mdl",
+    "models/hunter/triangles/1x05x05.mdl",
+    "models/props_c17/playgroundtick-tack-toe_block01a.mdl"
+}
+
+function cmd.cheese( caller, targets, shouldUncheese )
+    local prop = CFCUlxCommands.propify.propify( caller, targets, BASE_CHEESE_MODELS[ math.random( #BASE_CHEESE_MODELS ) ], shouldUncheese )
```

Comment:
If you move the cheese model into its own var above this one, this line will be very simple.
```lua
local model = BASE_CHEESE_MODELS[ math.blah
local prop = CFCUlxCommands.Propify.propify( caller, targets, cheeseModel, shouldUncheese )
```